scripts/main.py

```diff
@@ -1,8 +1,15 @@
 import sys
-from image import parse_image
+from PIL import Image
 
-image = parse_image(sys.argv)
+image = Image.open('images/image3.jpg')
 
-print(image)
+slices = []
 
-image.show()
\ No newline at end of file
+slice_height = int(image.height/10)
+slice_width = int(image.width/10)
+
+for x in range(0, 10):
+	for y in range(0, 10):
+		slices.append(image.crop((y*slice_width, x*slice_height, y*slice_width + slice_width, x*slice_height + slice_height)))
```

Comment:
You are losing the positions of slices by converting to an array of slices. You should keep a matrix of slices.

Reply:
Fixed